src/connectors/supla.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const filter = new MetadataFilter({
+	artist: cleanUpArtist,
+});
+
+Connector.playerSelector = 'main';
+
+Connector.artistTrackSelector = '.RadioHeader__WidgetContent-sc-17ofob1-4.irUZCR';
```

Comment:
I would go with a way we use in Plex connector.

https://github.com/web-scrobbler/web-scrobbler/blob/3db105c7508688bb88b5184939b09032a18854b9/src/connectors/plex.js#L9

For this particular property the selector could be like `[class^=RadioHeader__WidgetContent]`. The `irUZCR` should be omitted.

Reply:
Unfortunately `[class^=RadioHeader__WidgetContent]` doesn't work because it selects the program title of the radio channel. E.g. in the [Loop](https://www.supla.fi/loop) radio station page, the Web Scrobbler selects artist and track names from the title _Nyt: Loop iltapäivä - Paula "Paukkki" Rinta-Kanto_.

See an example photo below:
![kuva](https://user-images.githubusercontent.com/10089872/87327049-3bc99e00-c53c-11ea-973b-fa638220bf60.png)

Reply:
However, your tip works with the page that doesn't have a program title. Weird...
E.g. see the [Groove FM](https://www.supla.fi/groovefm) radio channel page:
![Näyttökuva 2020-7-13 kello 19 12 27](https://user-images.githubusercontent.com/10089872/87327503-cdd1a680-c53c-11ea-9fef-9916f2109ddb.png)

Reply:
Yes, the element matching the `[class^=RadioHeader__WidgetContent]` is not unique, so this suggestion won't work.